src/NLU.DevOps.ModelPerformance.Tests/TestCaseSourceTests.cs

```diff
@@ -390,8 +409,8 @@ public static void ToEntityTestCasesWithFalseNegativeEntityValueMismatch()
             int falsePositive,
             int falseNegative)
         {
-            var expectedUtterance = new LabeledUtterance(expected, null, null).WithProperty("speechFile", "speechFile");
-            var actualUtterance = new LabeledUtterance(actual, null, null);
+            var expectedUtterance = new LabeledUtterance(expected, string.Empty, null).WithProperty("speechFile", "speechFile");
```

Comment:
why does this need to change to empty string?

Reply:
Because, with this change, null intents throw an error.

---
In reply to: [379196304](https://github.com/microsoft/NLU.DevOps/pull/254#discussion_r379196304) [](ancestors = 379196304)